OamLog+forwardings.yaml

```diff
@@ -0,0 +1,518 @@
+        uuid: ol-1-0-0-op-s-bs-004
+      - service-name: /v1/regard-application
+        uuid: ol-1-0-0-op-s-is-001
+      - service-name: /v1/disregard-application
+        uuid: ol-1-0-0-op-s-is-002
+      - service-name: /v1/list-applications
+        uuid: ol-1-0-0-op-s-is-003
+      - service-name: /v1/record-oam-request
+        uuid: ol-1-0-0-op-s-is-004
+      - service-name: /v1/list-records
+        uuid: ol-1-0-0-op-s-is-005
+      - service-name: /v1/list-records-of-application
+        uuid: ol-1-0-0-op-s-is-006
+    consequent-requests:
+      - client-name: ExecutionAndTraceLog://v1/record-service-request
+        uuid: ol-1-0-0-op-c-bs-ol-1-0-0-000
+
+  - forwarding-name: OamRequestCausesLoggingRequest
+    uuid: ol-1-0-0-op-fc-bm-003
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: OamLog://v1/record-oam-request
+        uuid: ol-1-0-0-op-c-bs-ol-1-0-0-000
+
+  - forwarding-name: OamRequestCausesInquiryForAuthentication
+    uuid: ol-1-0-0-op-fc-bm-004
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/inquire-oam-request-approvals
+          uuid: ol-1-0-0-op-s-bm-005
+      fc-port-update:
+        - server-name: /v1/inquire-oam-request-approvals
+          uuid: ol-1-0-0-op-s-bm-005
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: AdministratorAdministration://v1/approve-oam-request
+        uuid: ol-1-0-0-op-c-bs-aa-1-0-0-000
+
+  - forwarding-name: PromptForBequeathingDataCausesTransferOfListOfApplications
+    uuid: ol-1-0-0-op-fc-im-110
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+      fc-port-update:
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      - server-name: /v1/bequeath-your-data-and-die
+        uuid: ol-1-0-0-op-s-im-000
+    consequent-requests:
+      - client-name: NewRelease:://v1/regard-application
+        uuid: ol-1-0-0-op-c-im-nr-1-0-0-000
+
+  - forwarding-name: PromptForBequeathingDataCausesTransferOfListOfRecords
+    uuid: ol-1-0-0-op-fc-im-111
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+      fc-port-update:
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      - server-name: /v1/bequeath-your-data-and-die
+        uuid: ol-1-0-0-op-s-im-000
+    consequent-requests:
+      - client-name: NewRelease:://v1/record-service-request
```

Comment:
the consequent-request to be updated to point to NewRelease://v1/record-oam-request, ol-1-0-0-op-c-im-nr-1-0-0-001

Reply:
Changed as proposed.